### DISCUSSION TITLE
Follow Requests

### DISCUSSION THREAD

**Discussion by metezd** [community/community#16555] [Other Feature Feedback, Questions, & Ideas]
Sending a request for following would be great. The user can decide who can follow. Like social platforms

People that don't follow you would see the follower and following number but they won't see who you follow or who follows you. Like Instagram.

An example for my profile's POV from a user that doesn't follow me

![Untitled](https://user-images.githubusercontent.com/37701679/168383091-fd11eee0-05d1-4865-bc3b-0ecfa317e31b.jpg)

**Comment by ghost** [comment]
@metezd Hi! How are you?  So... I had an idea similar to what you are asking for - reference: https://github.com/github/feedback/discussions/16751 - my idea would be to set the profile type to the GitHub user, so we would have 2 types of followers: public followers and private followers.

#### Concept 
- Public profile: anyone can follow you 
- Private profile: only people you want can follow you

**Comment by ghost** [comment]
I would like no one to follow me, I find this feature interesting. I think it's weird, people I don't have contact with follow me.
